smartling-files-api/src/main/java/com/smartling/api/files/v2/pto/UploadFileResponse.java

```diff
@@ -14,4 +14,5 @@
     private Integer stringCount;
     private Integer wordCount;
     private String message;
+    private String fileUri;
```

Comment:
please add it to the test
https://github.com/Smartling/java-api-sdk/blob/0bba5f7e65b331901769d659d9482a0ff5fdb846/smartling-files-api/src/test/java/com/smartling/api/files/v2/FilesApiIntTest.java#L91